documentation/development.md

```diff
@@ -0,0 +1,98 @@
+# Prerequisites
+1. Follow the [Setup Environment](setup-environment.md) wiki to setup your development environment.
+2. Follow the [Configuring a Stack](configuring-a-stack.md) wiki to configure your development stack settings.
+2. Follow the [Releasing](releasing.md) wiki to create a development release on the bucket's created in step 1.
+3. Follow the [Creating a New Stack](creating-a-new-stack.md) wiki to create a new development stack.
+
+# Development
+## Understanding Givesource Development
+Before you make any code changes, let's talk about what maybe unique with developing a serverless AWS application vs traditional web applications.
+
+1. Making changes to the frontend applications are all done locally, and should work like any other app.
+2. Making changes to the API may require:
+	- Updating lambda functions on AWS
+	- Updating / Adding / Deleting API resources or methods (endpoints). Changes must be applied through CloudFormation.
+	- Updating a backing service like a DynamoDb table. Changes must be applied through CloudFormation.
+
+_Understanding how Givesource uses AWS resources is just as important, if not more important, than understanding the codebase._
+
+## CloudFormation Development
+**Code Directory**:  
+`packages/cloudformation/templates`
+
+**Build Directory**:  
+`packages/cloudformation/build`
+
+**Making Changes**:  
+1. Verify your `.env` file is pointed to the correct Givesource stack.
+2. Verify the dependencies are up to date:
+	```
+	npm run frontend:install
+	```
+3. Build the templates:
+	```
+	npm run cloudformation:build
```

Comment:
same with these, probably worth noting to run these commands inside docker container, although not always necessary I guess

Reply:
The login and build are both referenced in the `Prerequisites` section. I'm fine expanding that or making it more explicit if we want to - but I was trying to avoid a complete re-write of this documentation